index.js

```diff
@@ -63,11 +78,14 @@ module.exports = {
       );
     }
 
-    // TODO: remove these `this.import` statements when the app is using using Embroider or ember-auto-import
-    this.import('vendor/qunit/qunit.js', { type: 'test' });
+    if (this._shouldImportQUnit) {
+      this.import('vendor/qunit/qunit.js', { type: 'test' });
+      this.import('vendor/ember-qunit/qunit-module.js', { type: 'test' });
+    }
+
+    // TODO: figure out how to make this not needed, AFAICT ember-auto-import
+    // does not provide any ability to import styles
```

Comment:
You are correct, but we really just need to enable that in ember-auto-import, because it's allowed in the embroider spec and we want ember-auto-import to be a faithful polyfill for that.

I think keeping the `app.import` here is fine until then, it's not harmful.